db/re/item_combo_db.txt

```diff
@@ -2526,3 +2526,46 @@
+490126:490127,{ bonus bFixedCast,-200; bonus bAspdRate,10; if (BaseLevel >= 105) { bonus2 bAddClass,Class_All,2; } if (BaseLevel >= 110) { bonus bShortAtkRate,5; bonus bLongAtkRate,5; } if (BaseLevel >= 120) { bonus bCritAtkRate,5; } }
+490128:490129,{ bonus bFixedCast,-200; bonus bVariableCastrate,-10; if (BaseLevel >= 105) { bonus bMatkRate,2; } if (BaseLevel >= 110) { bonus2 bMagicAtkEle,Ele_All,5; } if (BaseLevel >= 120) { bonus2 bMagicAddEle,Ele_All,5; } }
+15362:4123,{ bonus3 bAddEffWhenHit,Eff_Stun,5000,ATF_MAGIC; bonus3 bAddEffWhenHit,Eff_Freeze,5000,ATF_MAGIC; bonus3 bAddEffWhenHit,Eff_Stone,5000,ATF_MAGIC; }
+15362:4381,{ bonus bDelayrate,-5; bonus bAspd,1; }
+15367:4141,{ .@r = min(14,getequiprefinerycnt(EQI_ARMOR)); bonus bMaxHPrate,3*(.@r/2); }
+15367:4363,{ .@r = getrefine(); if (.@r>=10) { bonus bMaxHPrate,min(14,.@r)-5; } }
+15367:20905:22167,{ .@r = getequiprefinerycnt(EQI_ARMOR) + getequiprefinerycnt(EQI_GARMENT) + getequiprefinerycnt(EQI_SHOES); bonus bAllStats,3; bonus bMaxHPrate,15; bonus2 bSubSize,Size_Medium,5; if (.@r>=45) { bonus2 bResEff,Eff_Freeze,10000; } }
+15367:400075,{ bonus bHealPower,20; autobonus2 "{ bonus bDef,1000; bonus2 bSPLossRate,50,1000; }",20,5000,BF_WEAPON,"{ specialeffect2 EF_MAGNUMBREAK; }"; }
+15367:20905:22167:32219:400075,{ .@r = getequiprefinerycnt(EQI_HEAD_TOP) + getequiprefinerycnt(EQI_ARMOR) + getequiprefinerycnt(EQI_GARMENT) + getequiprefinerycnt(EQI_SHOES); if (BaseClass == Job_Arch_Bishop && .@r>=48) { skill "MG_ENERGYCOAT",1; } }
+19106:4132,{ bonus bMatkRate,10; bonus bUseSPrate,-50; bonus bHealPower,10; bonus2 bAddItemHealRate,526,15; bonus2 bAddItemSPHealRate,526,15; }
+19435:4001,{ bonus2 bResEff,Eff_Freeze,10000; }
+19217:4011,{ bonus bMaxHPrate,12; }
+19436:4145,{ bonus bVariableCastrate,30; }
+19436:4374,{ bonus2 bIgnoreMdefClassRate,Class_Boss,70; bonus2 bMagicAtkEle,Ele_Holy,30; bonus2 bMagicAtkEle,Ele_Neutral,30; }
+19436:32246,{ bonus bVariableCastrate,-20; }
+19443:4143,{ if (BaseLevel >= 100) { bonus bBaseAtk,30*(readparam(bVit)/10); bonus bMatk,30*(readparam(bVit)/10); bonus bVit,3*(readparam(bLuk)/10); bonus bLuk,3*(readparam(bLuk)/10); } else if (BaseLevel <= 99) { bonus bBaseAtk,10*(readparam(bVit)/10); bonus bMatk,10*(readparam(bVit)/10); bonus bVit,readparam(bLuk)/10; bonus bLuk,readparam(bLuk)/10; } }
+19444:4143,{ if (BaseLevel >= 100) { bonus bBaseAtk,30*(readparam(bVit)/10); bonus bMatk,30*(readparam(bVit)/10); bonus bVit,3*(readparam(bLuk)/10); bonus bLuk,3*(readparam(bLuk)/10); } else if (BaseLevel <= 99) { bonus bBaseAtk,10*(readparam(bVit)/10); bonus bMatk,10*(readparam(bVit)/10); bonus bVit,readparam(bLuk)/10; bonus bLuk,readparam(bLuk)/10; } }
+20816:4045,{ bonus bLongAtkDef,-35; }
+20816:28910,{ bonus bLongAtkDef,-35; }
+20905:22167,{ bonus bAllStats,1; bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus2 bSubRace,RC_All,10; bonus2 bSubRace,RC_Player_Doram,-10; bonus2 bSubRace,RC_Player_Human,-10; }
```

Comment:
For me the description include the player races in subrace all
```suggestion
20905:22167,{ bonus bAllStats,1; bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus2 bSubRace,RC_All,10; }
```

Reply:
Thanks